mayastor/src/core/env.rs

```diff
@@ -597,6 +597,13 @@ impl MayastorEnvironment {
         }
     }
 
+    pub(crate) fn import_nexuses() -> bool {
```

Comment:
I could not spot where this is used but -- Could we move this to src/subsys/config ?

Reply:
Moved there, where it's used